src/webui/webapplication.h

```diff
@@ -41,9 +41,9 @@
 #include "base/http/types.h"
 #include "base/utils/version.h"
 
-constexpr Utils::Version<int, 3, 2> API_VERSION {2, 0, 0};
+constexpr Utils::Version<int, 3, 2> API_VERSION {2, 0, 1};
 constexpr int COMPAT_API_VERSION = 18;
-constexpr int COMPAT_API_VERSION_MIN = 18;
+constexpr int COMPAT_API_VERSION_MIN = 19;
```

Comment:
Old API versioning was really confusing.
You need to change COMPAT_API_VERSION instead.